spec/jobs/appointment_notification/schedule_experiment_reminders_spec.rb

```diff
@@ -45,5 +45,18 @@
         described_class.perform_now
       }.to change { reminder.reload.status }.from("pending").to("scheduled")
     end
+
+    it "skips over reminders if there are failures, and sends other reminders" do
+      reminder_1 = create(:notification, subject: appointment, remind_on: today, status: "pending")
+      reminder_2 = create(:notification, subject: appointment, remind_on: today, status: "pending")
+
+      reminder_1.patient.discard
+
+      expect {
```

Comment:
I added this check for errors to ensure we hit an expectation error and not just an AR error, but this LGTM.